### ISSUE TITLE
Trial XXXXX failed: TypeError("step() missing 1 required positional argument: 'closure'")

### ISSUE THREAD

**Issue by oliver-lloyd** [uma-pi1/kge#275]
Hi,

I am repeatedly getting the titular error message on experiment trials. The only `step()` function I can find in LibKGE is that of the learning rate scheduler, here: https://github.com/uma-pi1/kge/blob/00de5127dd7fdc0d0d921a6dffda09a30dbd3d7d/kge/util/optimizer.py#L137

..but evidently that takes no 'closure' argument. Some googling suggests the issue might be to do with `torch.optim`, particularly the L-BFGS algorithm, but I am inexperienced with PyTorch and struggling to dig deeper.

I would greatly appreciate any guidance on how to fix this issue.

**Comment by rgemulla** [comment]
I cannot reproduce this error (numba 0.57.1, numpy 1.24.4, torch 2.0.1). Do you have a stack trace or any other additional information?

**Comment by oliver-lloyd** [comment]
I have no traceback unfortunately, but my package versions are earlier (numba 0.50.1, numpy 1.19.5, torch 1.7.1) so I will update and report back if the error reoccurs
